### PR TITLE
Upgrade TensorFlow

### PR DESCRIPTION
**Pull request by shendiaomo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ python:
 service:
 - docker
 install:
+- python -m pip install --upgrade pip
+- python -m pip install --upgrade setuptools>=41.0.0
+- python -m pip install --upgrade tensorflow==2.0.0
```
Comment (tonyyang-svail) [inline .travis.yml:17]: `tensorflow==2.0.0` will be installed by `step.py` via `python setup.py -q install`. Maybe remove it?

File: sqlflow_models/dnnclassifier.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def call(self, inputs, training=True):
             x = hidden_layer(x)
         return self.prediction_layer(x)
 
-def optimizer(learning_rate=0.001):
+def optimizer(learning_rate=0.01):
```
Comment (tonyyang-svail) [inline sqlflow_models/dnnclassifier.py]: Why update the default value in a package upgrading PR?

**Comment by tonyyang-svail** [comment]
Looks like the CI failed on the MaxCompute tests. Please file this PR from a branch of **this repo**, instead of your fork so that the secured environment variable can be accessed by CI.

File: .travis.yml (hunk 1 above)
Comment (shendiaomo) [reply .travis.yml:17]: I've tried what you say, but it's not viable. I think the `pip` and `setuptools` version in travis  is not up-to-date

File: sqlflow_models/dnnclassifier.py (hunk 2 above)
Comment (shendiaomo) [reply sqlflow_models/dnnclassifier.py]: There have many changes since TF2.0.0b1. Now the default learning rate of Adagrad is too small to stably meet the loss requirement of the unit test. (In fact, even in TF2.0.0b1, the unit test randomly fails to meet the loss criteria)
